test/coreneuron/test_datareturn.py

```diff
@@ -0,0 +1,195 @@
+# Test of data return covering most of the functionality.
```

Comment:
another side note/idea: coreneuron tests are starting to pile up. 
Would it make sense to cook up something similar to RxD with `pytest`?  What do you think @pramodk ?